src/n-dhcp4-c-probe.c

```diff
@@ -375,10 +373,8 @@ static void n_dhcp4_client_probe_config_initialize_random_seed(NDhcp4ClientProbe
  */
 uint32_t n_dhcp4_client_probe_config_get_random(NDhcp4ClientProbeConfig *config) {
         long int result;
-        int r;
 
-        r = mrand48_r(&config->entropy, &result);
-        c_assert(!r);
+        result = jrand48(config->entropy);
 
         return result;
```

Comment:
Why not just:

```c
        return jrand48(config->entropy);
```

Reply:
Oh, how did I not see that :D 

Thanks, fixed.